dev/com.ibm.websphere.appserver.features/visibility/private/io.openliberty.org.eclipse.microprofile.telemetry1.1.ee-9.0.feature

```diff
@@ -0,0 +1,18 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=io.openliberty.mpTelemetry1.1.ee-9.0
```

Comment:
This should be renamed to mp-5.0 instead of ee-9.0

Also you will need an mp-6.0 and mp-6.1 with the updates for those particular versions of MP.